mario.js

```diff
@@ -120,6 +120,7 @@
 
         if (windowsPhone) {
             detected.touch = t;
+            detected.windowsPhone = t;
```

Comment:
As I noted below, I think we should introduce some negative tests on this. So that we assert that desktop IE is NOT recognized as windowsPhone: true.